android-live-javacv/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea/workspace.xml
+/.idea/libraries
+.DS_Store
+/build
+/captures
+.externalNativeBuild
```

Comment:
I think you may want to gitignore everything in `.idea`? This is what we do in ExoPlayer, and it seems like it's good: https://github.com/muxinc/mux-stats-sdk-exoplayer/blob/master/.gitignore